proxy/src/main/java/net/md_5/bungee/conf/YamlConfig.java

```diff
@@ -91,11 +91,11 @@ public void load()
         {
             set( "permissions.default", Arrays.asList( new String[]
             {
-                "bungeecord.command.server", "bungeecord.command.list"
+                "bungeecord.command.server", "bungeecord.command.list", "bungeecord.command.find"
```

Comment:
I do not think that the find command should be available to everyone by default!